plugins/linguist-backend/README.md

```diff
@@ -0,0 +1,96 @@
+# Linguist Backend
+
+Welcome to the Linguist backend plugin! This plugin provides data for the Linguist frontend features.
+
+## Setup
+
+The following sections will help you get the Linguist Backend plugin setup and running.
+
+### Up and Running
+
+Here's how to get the backend up and running:
+
+1. First we need to add the `@backstage/plugin-linguist-backend` package to your backend:
+
+   ```sh
+   # From the Backstage root directory
+   cd packages/backend
+   yarn add @backstage/plugin-linguist-backend
+   ```
+
+2. Then we will create a new file named `packages/backend/src/plugins/linguist.ts`, and add the
+   following to it:
+
+    ```ts
+    import { TaskScheduleDefinition } from '@backstage/backend-tasks';
+    import { createRouter } from '@backstage/plugin-linguist-backend';
+    import { Router } from 'express';
+    import type { PluginEnvironment } from '../types';
+
+    export default async function createPlugin(
+      env: PluginEnvironment,
+    ): Promise<Router> {
+
+      const schedule: TaskScheduleDefinition = {
+        frequency: { hours: 24},
+        timeout: {minutes: 30},
+        initialDelay: {seconds: 90}
+      } 
+
+      return createRouter({
+        logger: env.logger,
+        config: env.config,
+        reader: env.reader,
+        discovery: env.discovery,
+        database: env.database,
+        permissions: env.permissions,
+        scheduler: env.scheduler,
+        schedule: schedule,
```

Comment:
Yes, I've played around with this as well and I'm not getting too far. So `env` in the `PluginEnvironment` which isn't exported so I'm not sure how to use that in my actual `createRouter` call. Could you maybe expand on this? Might help me understand better.

Reply:
Yeah I didn't mean to use the actual `PluginEnvironment` type in code, I just meant updating the README (without changing your code) to tell people that they out of practicality can do `...env` instead of the `a: env.a, b: env.b, .......` etc etc dance. As long as they named their env fields in the most common way, that equates to the same thing with less typing.

The second part of the comment was me thinking out loud that it might not be crazy to split the createRouter arguments into two, so that all of the env stuff is in the second object argument, and retaining only your plugin-specific things in the first object argument. Then the call would boil down to `createRouter({ pluginSpecificArg: value }, env)` for people who have standard naming of env fields. BUT you can ignore that last bit I think, the first ...env spread is plenty good. This is all just optional nitpicking anyway

Reply:
Ok, going to leave this for now. Just really want to get this in as it's getting pretty old :(

Reply:
Upon further thinking I've refactored this. Hopefully I've done this in the way you were thinking